src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -96,15 +96,14 @@ public Long getDurationInMinutes() {
         return Duration.between(initilizationDate, submissionDate).toMinutes();
     }
 
-    // TODO Ruscher, Entholzer: Refactor to getLatestResult
     /**
      * Is used as a workaround for objects that expect submission to have 1 result
      *
      * @return the latest result
      */
     @Nullable
-    @JsonProperty(value = "result", access = JsonProperty.Access.READ_ONLY)
-    public Result getResult() {
+    @JsonIgnore
+    public Result getLatestResult() {
```

Comment:
this returns the result with the highest `Id` right?

Reply:
Technically "latest" means the newest result? It might be great to add that information in the comment above

Reply:
We do not order the results by ID, we have a specific order_column which is handled by hibernate only. But yes, the latest result means the newest result